.chloggen/rnishtala_cw_receiver.yaml

```
@@ -0,0 +1,23 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiversawscloudwatch
```

```suggestion
component: awscloudwatchreceiver
```

I think this should be formatted like this right?

---

oh I meant, `receivers/awscloudwatch`